tests/core/test_jose/test_jwe.py

```diff
@@ -238,6 +251,135 @@ def test_ecdh_es_raise(self):
             protected, b'hello', key
         )
 
+    def test_ecdh_1pu_key_agreement_computation(self):
+        # https://tools.ietf.org/html/rfc7518#appendix-C
```

Comment:
Should it be https://datatracker.ietf.org/doc/html/draft-madden-jose-ecdh-1pu-04#appendix-A?